test/algorithms/test_spectrum.py

```diff
@@ -19,7 +19,7 @@
 )
 from toponetx.classes.cell_complex import CellComplex, CombinatorialComplex
 from toponetx.classes.simplicial_complex import SimplicialComplex
-from toponetx.datasets.mesh_complex import stanford_bunny
+from toponetx.datasets.mesh_datasets import stanford_bunny
```

Comment:
change to:
`toponetx.datasets.mesh`
(no need to repeat "datasets" after toponetx.datasets)